File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    
    count = 0
    
    for i in range(0, len(babbling)):
        babbling[i] = babbling[i].replace("aya", '*')
        babbling[i] = babbling[i].replace("**", 'aya')
        
        babbling[i] = babbling[i].replace("ye", '&')
        babbling[i] = babbling[i].replace("&&", 'ye')
        
        babbling[i] = babbling[i].replace("woo", '^')
        babbling[i] = babbling[i].replace("^^", 'woo')
        
        babbling[i] = babbling[i].replace("ma", '%')
        babbling[i] = babbling[i].replace("%%", 'ma')
    
    for j in range(0, len(babbling)):
        babbling[j] = babbling[j].replace("*", '')
        babbling[j] = babbling[j].replace("&", '')
        babbling[j] = babbling[j].replace("^", '')
        babbling[j] = babbling[j].replace("%", '')
        
        if(babbling[j] == ''):
            count += 1
        
        print(babbling[i])
    
    return count

File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.js
function solution(babbling) {
    count = 0;

    for (i = 0; i < babbling.length; i++) {
        babbling[i] = babbling[i].replace(/aya/g, '*');
        babbling[i] = babbling[i].replace(/\*\*/g, 'aya');
        
        babbling[i] = babbling[i].replace(/ye/g, '&');
        babbling[i] = babbling[i].replace(/&&/g, 'ye');
        
        babbling[i] = babbling[i].replace(/woo/g, '^');
        babbling[i] = babbling[i].replace(/\^\^/g, 'woo');
        
        babbling[i] = babbling[i].replace(/ma/g, '%');
        babbling[i] = babbling[i].replace(/%%/g, 'ma');
    }

    for (j = 0; j < babbling.length; j++) {
        babbling[j] = babbling[j].replace(/\*/g, '');
        babbling[j] = babbling[j].replace(/&/g, '');
        babbling[j] = babbling[j].replace(/\^/g, '');
        babbling[j] = babbling[j].replace(/%/g, '');
        
        if (babbling[j] === '') {
            count++;
        }
    }

    return count;
}